src/main/java/net/dv8tion/jda/internal/requests/restaction/MessageCreateActionImpl.java

```diff
@@ -95,6 +97,8 @@ protected RequestBody finalizeData()
                         .put("fail_if_not_exists", failOnInvalidReply)
                 );
             }
+            if (silent)
+                json.put("flags", json.getInt("flags", 0) | Message.MessageFlag.NOTIFICATIONS_SUPPRESSED.getValue());
```

Comment:
The `MessageCreateData` should already have flags

Reply:
Done.